lib/node_modules/@stdlib/string/next-grapheme-cluster-break/benchmark/benchmark.js

```diff
@@ -0,0 +1,54 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/random/base/randu' );
+var floor = require( '@stdlib/math/base/special/floor' );
+var fromCodePoint = require( '@stdlib/string/from-code-point' );
+var isNonNegativeInteger = require( '@stdlib/assert/is-nonnegative-integer' ).isPrimitive;
+var UNICODE_MAX = require( '@stdlib/constants/string/unicode-max' );
+var pkg = require( './../package.json' ).name;
+var nextGraphemeClusterBreak = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var out;
+	var x;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		x = floor( randu() * UNICODE_MAX );
```

Comment:
Instead of this approach, in order to get a better sense of runtime performance, we'd be better off choosing from a fixed set of strings. For example, before `b.tic()`, we can have a list of strings (they don't have to be long and can even be strings from tests and/or examples), and then at L43 we'd simply pick the next element (e.g., `list[ i%list.length ]`).